.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,211 @@
+      placeholder: docker-compose version 1.28.6, build 5db8d86f
+    validations:
+      required: false
+  - type: input
+    id: uname-movar
+    attributes:
+      label: The output of `uname -movar`
+      description: Please paste the output of `uname -movar` here. If you're unable to do so (maybe because you're on Windows), tell us the reason for your inability here.
+      placeholder: Linux swift 5.11.0-31-generic \#33-Ubuntu SMP Wed Aug 11 13:19:04 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux
+    validations:
+      required: true
+  - type: textarea
+    id: important-environment-variables
+    attributes:
+      label: Important environment variables
+      description: If there are environment variables important to this issue, please post them here. If we ask for the values of variables, make sure to put them in here too.
+      render: bash
+  - type: textarea
+    id: relevant-log-output
+    attributes:
+      label: Relevant log output
+      description: Show us relevant log output here. You can enable debug output by setting the environment variable `DMS_DEBUG` to `1`.
+      render: shell
+  - type: textarea
+    id: other-relevant-information
+    attributes:
+      label: Other relevant information
+      description: If there is more, you can tell us here. Make sure to use Markdown formatting here.
+      render: Markdown
+  - type: dropdown
+    id: experience
+    attributes:
+      label: What level of experience do you have with Docker and mail servers?
+      description: _You are not obliged to answer this question_. We do encourage answering though as it provides context to better assist you. Less experienced users tend to make common mistakes, which is ok; by letting us know we can spot those more easily. For the more experienced, we can skip ahead and communicate more effectively to resolve the issue ASAP.
+      options:
+        - I have no idea what I'm doing! [beginner]
+        - I know the basics, I've used Docker before [novice]
+        - I'm comfortable with Docker / containers, but I'm relatively new to all this mail stuff
+        - I have experience with mail servers, but less experience working with containers
+        - I know both pretty well actually [experienced]
+        - Trust me, I'm a (computer) engineer! [expert]
+    validations:
+      required: false
+  - type: checkboxes
+    id: terms-code-of-conduct
+    attributes:
+      label: Code of conduct
+      description: By submitting this issue, you agree to follow [our code of conduct](https://github.com/docker-mailserver/docker-mailserver/blob/master/CODE_OF_CONDUCT.md).
+      options:
+        - label: I read this project's Code of Conduct and I agree
+          required: true
+        - label: Just to make sure you're not just clicking checkboxes, please do not check this checkbox.
+          required: false
+        - label: I read the README and the documentation but could not find a solution
```

Comment:
```suggestion
        - label: I have read the README and the documentation but could not find a solution
```

Reply:
why the "have". Simple past is fine if you ask me :) It actually is, IIRC, better to use it here that present perfect because it implies a sense of having-already-done-it. But I guess this is about the fact that one cannot distinguish between simple present and simple past when reading "read". I will adjust this. (BTW this escalated quickly :D)

Reply:
Done. [998cb4b4884b4efcce5930e2c24d7cc902e658aa]